src/components/BoardHeader/ParticipantsList/ProgressCircle.tsx

```diff
@@ -0,0 +1,25 @@
+import {VFC} from "react";
```

Comment:
We should try to avoid `React.VFC` in the future because it's deprecated since React 18.0.0